Mta 1179 document assessment module (#759)
windup/windup-documentation: docs/topics/mta-assessment-module.adoc

Author: Will not publish this file.
Sections moved to other documents.

// Module included in the following assemblies:
//
// * docs/web-console-guide/master.adoc
// * topics/mta-assessment-module.adoc

:_content-type: REFERENCE
[id="mta-assessment-module_{context}"]
= Assessment Module

The assessment process for applications that are to be migrated in Red Hat Migration Toolbox for Applications (MTA) has been separated from other tool features and enhanced in version 7.0.0.

* A separate new Assessment hub includes the Assessment module, replacing the older _Pathfinder_ feature.
* Assessment supports multiple questionnaires, relevant to each assessed application.

> * Default quetionnaires can be cutomized and saved.

Reviewer: Typos
```suggestion
* Default questionnaires can be customized and saved.
```

// Module included in the following assemblies:
//
// * docs/web-console-guide/master.adoc
// * topics/mta-assessment-module.adoc

:_content-type: REFERENCE
[id="mta-assessment-module_{context}"]
= Assessment Module

The assessment process for applications that are to be migrated in Red Hat Migration Toolkit for Applications (MTA) has been separated from other tool features and enhanced in version 7.0.0.

* A separate new Assessment hub includes the Assessment module, replacing the older _Pathfinder_ feature.
* Assessment supports multiple questionnaires, relevant to each assessed application.
* Default questionnaires can be customized and saved.

> * User can design and version custom questionnaires and questions using YAML syntax.

Reviewer 2: ```suggestion
* Users can design and version custom questionnaires and questions using YAML syntax.
```

* Assessed applications are grouped into *archetypes* with similar application characteristics.
* Assessment module integrates tightly with the application inventory.
* Assessment questionnaires can be imported and exported.
* Enhanced questionnaire capabilities:
** Conditional questions
** Application auto-tagging according to answers
** Automated answers from tags in application archetypes

> 

Reviewer 3: Is the assessment feature new or renamed?

In the MTA user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved default and custom questionnaires.

> 

Reviewer 3: A module has only one type of cont. Before you submit this PR for peer review, be sure you've broken it down into modules that are called by "Include" statements in the master document.

:_content-type: PROCEDURE
[id="mta-assessment-start-questionnaire_{context}"]
== Opening an assessment questionnaire

> The {ProductName} ({ProductShortName}) assesses applications that are to be migrated according to a set of questions releavant to the application, such as dependencies, [more question types]. {ProductShortName} comes with a default questionnaire/s that the user can edit, customize and save as new questionnaires.

Reviewer: ```suggestion
The {ProductName} ({ProductShortName}) assesses applications that are to be migrated according to a set of questions relevant to the application, such as dependencies, [more question types]. {ProductShortName} comes with a default questionnaire/s that the user can edit, customize and save as new questionnaires.
```

> 

Reviewer 3: Why is [more question types] formatted in bold? is this a note to yourself? Please rewrite "a default questionnaire/s that the user can edit, customize and save as new questionnaires." Does the product come with more than one default questionnaire? Do we say "the user" or "you"? There should be a comma before "and" since customizing and saving are separate operations. Note that feature descriptions. There are a number of other problems with the text of this PR, but i'm going to focus on the visual aspects of it.

.To open a saved questionnaire

// Get updated image for MTA
image::images/assessment-process-start-01.png

. Open the {ProductName} user interface and click *Application Inventory* in the left menu bar. *Application assessment* appears in the main pane, listing all the available questionnaires. 
. Review all the questionnaires and click *Take* to the right of the selected questionnaire. The main pane shows a list of defined assessments.

// Get updated image for MTA
image::images/assessment-view-assessment-02.png

[start=3]

> . Right click one of the defined assessments and selct *view assessment*. The main pane shows the list of qustions in the assessment, separated into categories.

Reviewer: ```suggestion
. Right click one of the defined assessments and select *view assessment*. The main pane shows the list of questions in the assessment, separated into categories.
```

/ Get updated image for MTA
image::images/assessment-view-assessment-view-03.png

[start=4]

> . On the left of the main pane, click on a questions category. A list of questions for the category appears to the right of the categories list.
> . Click the down arrow to the left of one of the questions. A list of answer options appears below the list of questions.

Author: ```suggestion
. On the left of the main pane, click on a questions category. A list of questions for the category appears to the below of the categories list.
. Click the down arrow to the left of one of the rules. A list of answer options appears below the list of questions.
```

> . Select one of the nswers under *Answer choice*.

Reviewer: ```suggestion
. Select one of the answers under *Answer choice*.
```
